docs/configuration.md

```diff
@@ -272,3 +273,43 @@ You shall read output like the following:
 OK
 ```
 If the file is valid YAML but does not contain a valid configuration, it prints a helpful output.
+
+
+## Database backend
+
+Aeternity nodes support several types of database persistence backends:
+
+ - RocksDB (default for Unix, available for Unix)
+ - Mnesia (default for Win32, available for all)
+ - Leveled (experimental, available for all)
+
+You may choose the database backend by setting `chain.db_backend` to the corresponding value `rocksdb`, `mnesia`, `leveled`
+
+Example:
+
+```yaml
+chain:
+    persist: true
+    db_path: ./my_db
+    db_backend: leveled
+```
+
+**RocksDB** is only available under Unix compatible systems (including OS X and WSL) and is used there by default.
+RocksDB does not work with NTFS volumes.
+
+**Leveled** is designed to be a better alternative to RocksDB and is available for all OS'es.
+However currently it has an experimental support.
+
+**Mnesia** is the DB backend that is distributed with Erlang/OTP but is considered less reliable than the other two.
```

Comment:
I wouldn't say so. Technically mnesia is also used when choosing the other 2 db backends. Just that the mnesia disk storage is not used. It has some drawbacks compared to the other options, but I don't think reliability is it.

Reply:
I probably didn't use the proper term, may be less performant?